RELEASE-NOTES.md

```diff
@@ -1,8 +1,97 @@
+- Deployments created from served flows will have a `READY` status if its associated process is running.
+- Deployments created in a work pool will have a `READY` status when a worker is `ONLINE` and polling the associated work queue.
+- Deployments created in a push work pool (Prefect Cloud) will always have a `READY` status.
+  
+<p align="center">
+<img width="976" alt="a late flow run for a deployment that is `NOT_READY`" src="https://github.com/PrefectHQ/prefect/assets/42048900/db20979a-870d-44c4-ac0b-66f70d99e58b">
+</p>
+
+In Prefect Cloud, an event is emitted each time a deployment changes status. These events are viewable in the Event Feed.
+<p align="center">
+<img width="538" alt="event feed deployment status events" src="https://github.com/PrefectHQ/prefect/assets/42048900/8ee076cd-fd30-47d1-9ee5-6b5a3b383b63">
+</p>
+
+You can also create an automation triggered by deployment status changes on the Automations page!
+
+<p align="center">
+<img width="862" alt="deployment status trigger on automations page" src="https://github.com/PrefectHQ/prefect/assets/42048900/87a0945e-9b9e-406b-b020-fbd9733cb4c3">
+</p>
+
+### Additional storage options for `flow.from_source`
+
+You can now load flows from a variety of storage options with `flow.from_source`! In addition to loading flows from a git repository, you can load flows from any supported `fsspec` protocol.
+
+Here's an example of loading and serving a flow from and S3 bucket:
+
+```python
+from prefect import flow
+
+if __name__ == "__main__":
+    flow.from_source(
+        source="s3://my-bucket/my-folder",
+        entrypoint="flows.py:my_flow",
+    ).serve(name="deployment-from-remote-flow")
+```
+
+You can use the `RemoteStorage` class to provide additional configuration options.
+
+Here's an example of loading and serving a flow from Azure Blob Storage with a custom account name:
+
+```python
+from prefect import flow
+from prefect.runner.storage import RemoteStorage
+
+if __name__ == "__main__":
+    flow.from_source(
+        source=RemoteStorage(url="az://my-container/my-folder", account_name="my-account-name")
+        entrypoint="flows.py:my_flow",
+    ).serve(name="deployment-from-remote-flow")
+```
+
+See the following pull request for implementation details:
+- https://github.com/PrefectHQ/prefect/pull/11072
+
+### Enhancements
+- Add option to skip building a Docker image with `flow.deploy` — https://github.com/PrefectHQ/prefect/pull/11082
+- Add variables page empty state — https://github.com/PrefectHQ/prefect/pull/11044
+- Allow composite sort of block documents by `block_type_name` and name — https://github.com/PrefectHQ/prefect/pull/11054
+- Warn if task parameter introspection takes a long time — https://github.com/PrefectHQ/prefect/pull/11075
```

Comment:
This actually also adds a new setting, which might make sense to mention

Reply:
i did see that, i didnt want to make the line too long but lets see what i can do